inc/saitam.h

```diff
@@ -1784,6 +1784,135 @@ typedef sai_status_t (*sai_set_tam_collector_attribute_fn)(
         _In_ sai_object_id_t tam_collector_id,
         _In_ const sai_attribute_t *attr);
 
+/**
+ * @brief TAM Packet Drop Types
+ */
+typedef enum _sai_packet_drop_type_t
+{
```

Comment:
L3_NEXTHOP DROP is version import for us,  could you implement it

Reply:
> L3_NEXTHOP DROP is very important for us, could you implement it

Yes, will add it